src/OpenTelemetry.Instrumentation.Runtime/README.md

```diff
@@ -64,9 +64,13 @@ complete demo.
 
 ### GC related metrics
 
-| Name                                            | Description                                                           | Units           | Instrument Type   | Value Type | Attribute Key(s) | Attribute Values |
-|-------------------------------------------------|-----------------------------------------------------------------------|-----------------|-------------------|------------|------------------|------------------|
-| process.runtime.dotnet.**gc.collections.count** | Number of garbage collections that have occurred since process start. | `{collections}` | ObservableCounter | `Int64`    | generation       | gen0, gen1, gen2 |
+#### process.runtime.dotnet.**gc.collections.count**
+
+| Units           | Instrument Type   | Value Type | Attribute Key(s) | Attribute Values |
+|-----------------|-------------------|------------|------------------|------------------|
+| `{collections}` | ObservableCounter | `Int64`    | generation       | gen0, gen1, gen2 |
+
+Description: Number of garbage collections that have occurred since process start.
```

Comment:
I suggest that we make the description the 1st sentence of each metric.

Reply:
Done.

Reply:
@xiang17 could you also remove the "Description: " part? It doesn't seem to add value here.